Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "review-web"
-version = "0.20.0"
+version = "0.21.0-alpha.1"
```

Comment:
@sophie-cluml @BLYKIM 
I believe alpha versions imply they are ready to test for a new release. However, because we still have to do much work until this is so, I don't think this alpha is appropriate. I really appreciate it if you feel free to share your thoughts on this.

Reply:
제가 이해하는 바로는 alpha와 같은 pre-release는 다음 상태를 의미합니다. 

> A pre-release version indicates that the version is unstable and might not satisfy the intended compatibility requirements as denoted by its associated normal version.

이번 PR에서는 review-db의 스키마 변경이 있어서, 그 결과로, 두가지 큰 변화가 발생한다고 보고 있습니다. 하나는 review-db 스키마 변경으로 촉발되는 node 관련 GraphQL API 변경이고, 다른 하나는 이 리포지토리 자체의 실행모듈인 minireview를 구동했을 때, review-db의 migration이 발생하면서, 더 이상 과거의 commit으로 되돌아가서 minireview를 구동할 수 없는 상황이 됩니다. 저는 이 둘 중 하나라도 발생하면 version up을 하면서 alpha 버전을 찍는 것으로 이해하고 있습니다. (우리가 기존에 review-web에서 과거에 어떻게 해왔느냐와는 별개로 이 방향이 나아가야할 방향으로 이해하고 있습니다)

말씀주신 사항 중 앞으로 할 일이 많이 남아있다는 점은 제가 이해한 바로는 pre-release 를 찍지 않아야 할 조건으로 고려하고 있는 것은 아닌걸로 알고 있습니다. review-database를 사례로 하여 이해하고 있는데요, review-database에서는 스키마 변경이 생기면, review-database가 다음 릴리즈까지 남아있는 추가작업의 양과 독립적으로 alpha 버전을 찍는 것으로 알고 있습니다. 저는 이러한 방식으로 관리하는 방식이 review-database가 라이브러리로서 자신을 사용하는 클라이언트들을 위하여 version up을 하면서 pre-release로 alpha 버전 넘버링을 하는 방식으로 이해하고 있는 상황인데요, review-web과 review-web의 API를 호출하는 UI 클라이언트 리포지토리 간의 관계와, review-web과 minireview 간의 관계는 review-database와 review-web 간의 관계에 대응된다고 생각하고 있었습니다.

이 부분 제가 잘못 알고 있을 수 있으니 알려주시면 감사하겠습니다. 사실 기계적으로 할 수 있는 기준이 있다면 더 좋을 것 같아서 이번 기회에 가이드를 주시면 매우 좋을 것 같습니다. 알려주시면 저희가 관리하고 있는 "버전 규칙" 쪽에 적어볼 수 있을 것 같아요. 그리고 만약 좀 더 면밀한 디스커션이 필요하다면 이번 PR은 sehkone 의 제안대로 하고 따로 sync up을 하는 기회를 갖는 것도 가능할까요? 

cc. @BLYKIM

Reply:
As you pointed out, any changes to the database following its migration require an alpha version as well. However, I'm not sure if this pertains to this review-web, since the review-database only manages the database. I don't understand your comment regarding the mini-review. Could you elaborate on it? I don't see any cases where the mini-review requires versioning of review-web.

Reply:
별도 디스커션을 통해 다음과 같은 관점에서 바라보는 것으로 정리되었습니다. 정리된 내용이 @sehkone 이 이해하고 있는 바와 다르다면 알려주세요.

- review-web이 "릴리즈" 되면 management server application 는 언제든 릴리즈할 준비가 되어있는 경우가 대다수이다.
- review-web은 review-web을 라이브러리로 사용하는 management server application을 alpha버전으로 "프리릴리즈" 하고자 할 때 사용한다. 그리고 이는 거의 발생하지 않을 상황이다.

@BLYKIM 위와 같은 방향으로 sync 가 되신다면 이번 PR에서는 alpha 버전을 제거하는 게 맞는 방향이어서 변경을 부탁드리고자 합니다. 🙇

Reply:
done

Reply:
> review-web은 review-web을 라이브러리로 사용하는 management server application을 alpha버전으로 "프리릴리즈" 하고자 할 때 사용한다. 그리고 이는 거의 발생하지 않을 상황이다.

이 말은 제가 잘 이해를 못 했습니다. 제가 나름대로 고쳐보면 아래가 아닌가 추측합니다.
review-web**의 alpha버전은** review-web을 라이브러리로 사용하는 management server application을 alpha버전으로 "프리릴리즈" 하고자 할 때 사용한다. 그리고 이는 거의 발생하지 않을 상황이다.